lib/events/roomcreateevent.cpp

```diff
@@ -5,6 +5,26 @@
 
 using namespace Quotient;
 
+static const std::array<QString, 1> roomTypeStrings = {
+    { "m.space" }
+};
```

Comment:
The CI reminds that VS 2019 still doesn't support that kind of things.
```suggestion
constexpr auto roomTypeStrings = to_array({
    { "m.space" }
});
```
(`Quotient::to_array` is defined in `quotient_common.h`, while compilers are catching up with C++20 support.)

Also, if you decide on moving out the `RoomType` enumeration you'll have to move it over to `quotient_common.h` because `Q_ENUM_NS()` only works where `Q_NAMESPACE` is, and `Q_NAMESPACE` should occur exactly once for a namespace.

Reply:
...and I'm stupid, it's not about the support, it's about `#include <array>` needed for MS STL but not for other standard libraries. Anyway, `to_array()` is a good facility.